test/test_board.py

```diff
@@ -75,3 +75,69 @@ def test_is_spot_taken_returns_false_with_multiple_spots_taken_2(self):
         user_input = 6
         spot_is_taken = board.is_spot_taken(current_board, user_input)
         self.assertEqual(False, spot_is_taken)
+
+    def test_if_three_in_horizontal_first_row_return_true(self):
+        board = Board()
+        current_board = ["X", "X", "X", "O", "5", "6", "7", "8", "9"]
+        winner = board.is_winner_horizontal(current_board)
+        self.assertEqual(True, winner)
```

Comment:
This and the next 11 tests all follow the same pattern.  Do you know if Python allows for parameterized tests?  Have you heard of them?  It may allow for you do something LIKE:

```
{
  test_board: ["X", "X", "X", "O", "5", "6", "7", "8", "9"], expected_result: True,
  test_board: ["1", "2", "3", "X", "X", "X", "7", "8", "9"], expected_result: True,
  ...
  test_board :["X", "2", "X", "4", "X", "6", "7", "8", "9"], expected_result: False
}
def parmtest_winning_scenarios(self):
        board = Board()
        winner = board.is_winner_horizontal(test_board)
        self.assertEqual(expected_result, winner)
```

Reply:
I have found some articles but I'll need to take some time to research it.  Can I put this on the back burner to work on the tickets and merge this without this for now?

Reply:
totally